### ISSUE TITLE
propose new function Cmi5.moveOn (or other name)?

### ISSUE THREAD

**Issue by beatthat** [xapijs/cmi5#27]
Most of the time when we make an assignable unit, the devs don't care about xapi or cmi5 and they just want to know how to send results when their lesson content is complete.

Cmi5 on the other side, has a whole end sequence for assignable units and it needs to be done correctly and completely.

So to simplify for the above use case, it would be good to have a function like `moveOn` that handles all the COMPLETED, `PASSED`|`FAILED` and terminated messages the boilerplate way for the caller, something like:

```typescript
interface MoveOnOptions extends ... {
  disableSendTerminate: boolean;
}

async function moveOn(score: number|ResultScore, options: MoveOnOptions): Promise<string[]> {
  // check launchData moveOn and masteryScore and send COMPLETED and/or PASSED|FAILED
  // send TERMINATED unless disabled in params
}
```

...if this sounds good, I can send as a PR

**Comment by CookieCookson** [comment]
I like the sound of this, it reduces a lot of boilerplate code for the developer to implement and ensures it is s performed in the correct sequence. Happy for you to submit a PR as described!

**Comment by beatthat** [comment]
great. PR is [here](https://github.com/xapijs/cmi5/pull/50)

I can't request review directly

**Comment by CookieCookson** [comment]
Hi @beatthat thanks for taking the time to put a PR together. I've been a bit snowed under with things outside of work lately, but I hope to give this a serious look towards the weekend.

**Comment by CookieCookson** [comment]
Thanks for the contribution @beatthat, your new feature has been merged into v0.3.0!
